scripts/sct_compute_mtr.py

```diff
@@ -141,6 +155,12 @@ def get_parser():
                       mandatory=False,
                       example=['0', '1', '2'],
                       default_value='1')
+    parser.add_option(name="-omtr",
+                      type_value="str",
+                      description="Creates output file with the specified path.",
+                      mandatory=False,
+                      example='My_File_Folder/My_New_File',
```

Comment:
give more concrete example, e.g.: `bob/data/mtr.nii.gz`

Reply:
agreed, fixed in commit [5052b25](https://github.com/YangHee-Min/spinalcordtoolbox/commit/5052b25c7044ccd53eb796f4af44dbafeff86f85)